### PR TITLE
Sort by name

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

**Comment by topless** [comment]
My initial thought, but I am kind of "biased", maybe I ll sort everything but the tribe should come first.

**Comment by lipis** [comment]
Well.. make your biased product the best in the world.. and it won't be an issue :D

**Comment by lipis** [comment]
Get rid of the `The` and you'll be on top of the list again..

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
+a
```
Comment (lipis) [inline README.md:1]: fail :)

**Pull request merged**